iOSNativeSwiftTemplate/iOSNativeSwiftTemplate/SceneDelegate.swift

```diff
@@ -109,4 +139,34 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate {
        }
        postResetBlock()
    }
+    
+    // MARK: - QR Code Login Via Salesforce Identity API UI Bridge Private Implementation
+    
+    /**
+     * Validates and uses a QR code log in URL.
+     * - Parameters
+     *   - url: The URL to validate and use as a QR code log in URL
+     */
+    private func useQrCodeLogInUrl(_ url: URL) {
+        
+        // Use login QR code if applicable
+        
+        // Validate the URL is a valid QR code login URL.
+        guard let components = NSURLComponents(
+            url: url,
+            resolvingAgainstBaseURL: true),
+              let scheme = components.scheme,
+              scheme == "mobileapp",
+              let host = components.host,
+              host == "android",
```

Comment:
are we using android as the example for both platforms?

Reply:
I pondered changing it here, even though the actual string value in the template is arbitrary and should be changed by the template user.  If we did want to change it, we could follow-up with pull requests for both Android and iOS to pick a common value and then ensure the reference Apex/VIsualforce Page match.

What if we pulled a page from our other recent authentication efforts and put in a placeholder value that is followed by an assertion requiring it have been changed?

Reply:
@bbirman - This looks like a winner, though it highlights that setting up QR code log in is bit complicated. 
https://github.com/forcedotcom/SalesforceMobileSDK-Templates/pull/427/commits/7833e403f7eeba12f3e92539a406c86c59c2905b

@brandonpage & @wmathurin, if this looks good on the iOS side I reckon a quick commit to the Android template to match would be really helpful.

Reply:
👍 we should use a place holder value, assert it get replaced, document it needs to match the server side.
In 12.2, the apex page will be in the doc, right?

Reply:
Correct on both ☝🏻